custom_components/ha_washdata/ml/engine.py

```
@@ -52,6 +52,61 @@
+    ``feature_extraction``) must already be in ``sys.modules`` by then - Home
+    Assistant flags a blocking ``importlib.import_module`` in the loop (issue
+    #328). Also warms the manifest cache, which reads a file. Never raises;
+    idempotent, so calling it once per config entry is cheap.
+    """
+    for module_name in _MODEL_MODULES.values():
+        _load_model_module(module_name)
+    for sibling in _SIBLING_MODULES:
+        try:
+            importlib.import_module(f"{__package__}.{sibling}")
+        except Exception as exc:  # noqa: BLE001 - best effort warm-up
+            _LOGGER.warning("Failed to preload ML module %r: %s", sibling, exc)
```

Ruff (0.16.1)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)